crates/typos-cli/src/bin/typos-cli/main.rs

```diff
@@ -316,6 +317,13 @@ fn run_checks(args: &args::Args) -> proc_exit::ExitResult {
         if status_reporter.errors_found() {
             errors_found = true;
         }
+        
+        match status_reporter.finalize() {
+            Ok(_) => {}
+            Err(err) => {
+                log::error!("Error finalizing: {}", err);
```

Comment:
`Error` would be redundant with log showing `ERROR`.  `finalizing` has no user-facing meaning in this context

Reply:
updated

Reply:
`generate final result` still doesn't have much meaning to the user and reads oddly within context of an error.

Likely we should directly print the error and have `error_to_io_error` change the error message to `format!("failed to generate SARIF output: {err}")`

Reply:
updated

Reply:
Not seeing this updated